api/cmd/transformer/main.go

```diff
@@ -60,6 +63,18 @@ type AppConfig struct {
 
 	// By default the value is 0, users should configure this value below the memory requested
 	InitHeapSizeInMB int `envconfig:"INIT_HEAP_SIZE_IN_MB" default:"0"`
+
+	Tracing JaegerTracing
+}
+
+// JaegerConfig holds configuration about jaeger tracing
+type JaegerTracing struct {
+	// Disabled the tracing, will create NoOpTracerProvider
+	Disabled bool `envconfig:"JAEGER_DISABLED" default:"true"`
+	// Collector endpoint URL
+	CollectorURL string `envconfig:"JAEGER_ENDPOINT"`
```

Comment:
Can we explicitly use `JAEGER_COLLECTOR_URL` instead of calling it `JAEGER_ENDPOINT` ?

Reply:
Can you remove unused JAEGER constants under constant.go?

Reply:
updated